lib/internal/test_runner/runner.js

```diff
@@ -402,7 +402,8 @@ function runTestFile(path, filesWatcher, opts) {
       throw err;
     }
   });
-  return subtest.start();
+  const subTestEnded = subtest.start();
```

Comment:
Why did you make this change? Can you add a comment?

Reply:
so `subTestEnded` can be awaited? I replaced the `queueMicrotask` with waiting for the test to complete (see line 385)